src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -110,6 +110,7 @@ public void DynamicDirectory_Null()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Not reliable when multiple executable assemblies are in the output directory.")]
```

Comment:
Do we have an issue logged for this?

Reply:
thanks, removed the skip as it doesn't throw anymore.